src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
@am11, I am all for extensibility, but just making types public does not ensure that extensibility is suddenly enabled, and making some types internal does not mean it's not possible. Please tell me what scenarios are disabled because these types are made internal?